prov/util/src/util_attr.c

```diff
@@ -445,6 +448,9 @@ int ofi_check_mr_mode(uint32_t api_version, uint32_t prov_mode,
 			return -FI_ENODATA;
 		}
 	} else {
+		if (!(caps & FI_RMA))
+			prov_mode &= ~OFI_MR_BASIC_MAP;
```

Comment:
Need FI_ATOMIC as part of the check.  I think we can discard FI_MR_PROV_KEY and FI_MR_VIRT_ADDR.  We can also discard FI_MR_ALLOCATED if FI_MR_LOCAL is not required.